rules/macos/execution_scripting_osascript_exec_followed_by_netcon.toml

```
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects AppleScript execution via the Apple Scripting Interpreter (osascript) followed by a network connection from
+the same process within a short time period. Adversaries may abuse AppleScript for execution and command and
+control.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "AppleScript Execution followed by Network Connection"
```

```suggestion
name = "Apple Script Execution followed by Network Connection"
```
see comment above. changing to Apple Script since AppleScript is language specific